### ISSUE TITLE
The BlockBreadcrumb shouldn't be a NavigableToolbar

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6337]
The `BlockBreadcrumb` component added in #5658 is wrapped in a `NavigableToolbar` component.

Seems to me this is not appropriate because:
- NavigableToolbar adds arrows navigation via NavigableMenu
- it also adds a `role="toolbar"` which is not appropriate in this case.

Maybe I'm missing something but since `BlockBreadcrumb` is not focusable and can't be used with a keyboard, it should use any of that. If it's supposed to be a mouse-only feature, then it should be just a div without an ARIA role. Also, arrow navigation is pointless.

See also previous comment on the nested Cover Image PR https://github.com/WordPress/gutenberg/pull/5452#issuecomment-383053113 where I'm suggesting to add the "Select parent block" button to the real block toolbar.

/Cc @jasmussen @jorgefilipecosta
